### ISSUE TITLE
Rename to language-carto

### ISSUE THREAD

**Issue by mnquintana** [yohanboniface/carto-atom#2]
It would be great if you could republish this package as `language-carto` to better fit in with the conventions for naming Atom grammar packages. See https://github.com/atom/atom/blob/master/docs/apm-rest-api.md#renaming-a-package

You wouldn't need to change the name of the repo though.

Thanks for making this!

**Comment by yohanboniface** [comment]
Oops, sorry, missed the notification, as always with the messy Github notifications :/
I'll have a look asap :)

**Comment by mnquintana** [comment]
Although that doc doesn't say it, you shouldn't need to interact with the REST API directly to rename it - you should just be able to rename it in `package.json` and then release a new version with `apm publish minor`, I think.

**Comment by yohanboniface** [comment]
I may be to late at the party:

```
$ apm publish minor
Registering language-carto ✗
Registering package in yohanboniface/carto-atom repository failed: A package by that name already exists
```

**Comment by yohanboniface** [comment]
@mnquintana I've added you as collaborator, feel free to do what's needed :)

**Comment by mnquintana** [comment]
Thanks @yohanboniface! Just published 0.6.0 as language-carto: https://atom.io/packages/language-carto

**Comment by yohanboniface** [comment]
Thank you :)
